filebeat/module/apache/access/ingest/default.json

```
@@ -4,6 +4,7 @@
     "grok": {
       "field": "message",
       "patterns":[
+        "%{IPORHOST:http.vhost} %{IPORHOST:source.address} - %{DATA:user.name} \\[%{HTTPDATE:apache.access.time}\\] \"(?:%{WORD:http.request.method} %{DATA:url.original} HTTP/%{NUMBER:http.version}|-)?\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)( \"%{DATA:http.request.referrer}\")?( \"%{DATA:user_agent.original}\")?",
```

To follow [ECS](https://www.elastic.co/guide/en/ecs/1.0/ecs-field-reference.html), I think that the vhost should go to the `source.domain`.
And I wonder if in this case we can consider that the source address is always going to be an IP, then we could directly store it in `source.ip`.
Do you know of any case where both fields are domains?

```suggestion
        "%{IPORHOST:source.domain} %{IPORHOST:source.ip} - %{DATA:user.name} \\[%{HTTPDATE:apache.access.time}\\] \"(?:%{WORD:http.request.method} %{DATA:url.original} HTTP/%{NUMBER:http.version}|-)?\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)( \"%{DATA:http.request.referrer}\")?( \"%{DATA:user_agent.original}\")?",
```

---

the second field is never a domain

---

Eh, how is the vhost anything to do with the source? It's an attribute of the **server**...